files/etc/uci-defaults/99-nethsec-default-distfeed

```diff
@@ -0,0 +1,6 @@
+[ "$(uci -q get ns-plug.config.repository_url)" != "" ] && exit 0
+
+uci -q batch << EOI
+set ns-plug.config.repository_url="https://updates.nethsecurity.nethserver.org"
+commit ns-plug
+EOI
```

Comment:
Move the implementation inside [20_ns-plug script](https://github.com/NethServer/nethsecurity/blob/main/packages/ns-plug/files/20_ns-plug), before line 9:
- the url must be set even when updating with packages, the 20_ns-plug is [already executed on update](https://github.com/NethServer/nethsecurity/blob/main/packages/ns-plug/Makefile#L40-L48)
- the url must be set before invoking distfeed